src/App/components/ActionList/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+import { Icon } from "#";
+
+const ActionList = ({ id, classNames, toggleBtn, items }) => (
+    <>
+        <div id={id} className={`action-list${classNames ? ` ${classNames}` : ""}`}>{"\n"}
+            {toggleBtn ? <>{toggleBtn}{"\n"}</> : <>{<i className="material-icons action-toggle">more_vert</i>}</>}
+            <div className={`action-menu${classNames ? ` ${classNames}` : ""}`}>{"\n"}
+                {
+                    (Array.isArray(items)) ? items.map(({ name, icon }, i) => (
```

Comment:
You already have PropTypes checking this, have some faith in your users

```suggestion
                    items.map(({ name, icon }, i) => (
```